src/main/java/me/hydos/lint/client/entity/model/KingTaterEntityModel.java

```diff
@@ -21,106 +21,103 @@
-		back.addCuboid("back", 6.5F, -35.0F, 11.0F, 1, 1, 0, 0.0F, 8, 77);
-		back.addCuboid("back", 4.5F, -35.0F, 11.0F, 1, 1, 0, 0.0F, 8, 77);
-		back.addCuboid("back", 2.5F, -35.0F, 11.0F, 1, 1, 0, 0.0F, 8, 77);
-		back.addCuboid("back", 0.5F, -35.0F, 11.0F, 1, 1, 0, 0.0F, 8, 77);
-		back.addCuboid("back", -1.5F, -35.0F, 11.0F, 1, 1, 0, 0.0F, 8, 77);
-		back.addCuboid("back", -3.5F, -35.0F, 11.0F, 1, 1, 0, 0.0F, 8, 77);
-		back.addCuboid("back", -5.5F, -35.0F, 11.0F, 1, 1, 0, 0.0F, 8, 77);
-		back.addCuboid("back", -7.5F, -35.0F, 11.0F, 1, 1, 0, 0.0F, 8, 77);
-		back.addCuboid("back", -9.5F, -35.0F, 11.0F, 1, 1, 0, 0.0F, 8, 77);
-		back.addCuboid("back", -11.5F, -35.0F, 11.0F, 1, 1, 0, 0.0F, 8, 77);
-
-		ModelPart right = new ModelPart(this);
-		right.setPivot(0.0F, 0.0F, 0.0F);
-		things.addChild(right);
-		right.addCuboid("right", 12.0F, -35.0F, -11.5F, 0, 1, 1, 0.0F, 31, 68);
-		right.addCuboid("right", 12.0F, -35.0F, -9.5F, 0, 1, 1, 0.0F, 31, 68);
-		right.addCuboid("right", 12.0F, -35.0F, -7.5F, 0, 1, 1, 0.0F, 31, 68);
-		right.addCuboid("right", 12.0F, -35.0F, -5.5F, 0, 1, 1, 0.0F, 31, 68);
-		right.addCuboid("right", 12.0F, -35.0F, -3.5F, 0, 1, 1, 0.0F, 31, 68);
-		right.addCuboid("right", 12.0F, -35.0F, -1.5F, 0, 1, 1, 0.0F, 31, 68);
-		right.addCuboid("right", 12.0F, -35.0F, 0.5F, 0, 1, 1, 0.0F, 31, 68);
-		right.addCuboid("right", 12.0F, -35.0F, 2.5F, 0, 1, 1, 0.0F, 31, 68);
-		right.addCuboid("right", 12.0F, -35.0F, 4.5F, 0, 1, 1, 0.0F, 31, 68);
-		right.addCuboid("right", 12.0F, -35.0F, 6.5F, 0, 1, 1, 0.0F, 31, 68);
-		right.addCuboid("right", 12.0F, -35.0F, 8.5F, 0, 1, 1, 0.0F, 31, 68);
-		right.addCuboid("right", 12.0F, -35.0F, 10.5F, 0, 1, 1, 0.0F, 31, 68);
-
-		ModelPart left = new ModelPart(this);
-		left.setPivot(0.0F, 0.0F, 0.0F);
-		things.addChild(left);
-		left.addCuboid("left", -12.0F, -35.0F, -11.5F, 0, 1, 1, 0.0F, 31, 68);
-		left.addCuboid("left", -12.0F, -35.0F, -9.5F, 0, 1, 1, 0.0F, 31, 68);
-		left.addCuboid("left", -12.0F, -35.0F, -7.5F, 0, 1, 1, 0.0F, 31, 68);
-		left.addCuboid("left", -12.0F, -35.0F, -5.5F, 0, 1, 1, 0.0F, 31, 68);
-		left.addCuboid("left", -12.0F, -35.0F, -3.5F, 0, 1, 1, 0.0F, 31, 68);
-		left.addCuboid("left", -12.0F, -35.0F, -1.5F, 0, 1, 1, 0.0F, 31, 68);
-		left.addCuboid("left", -12.0F, -35.0F, 0.5F, 0, 1, 1, 0.0F, 31, 68);
-		left.addCuboid("left", -12.0F, -35.0F, 2.5F, 0, 1, 1, 0.0F, 31, 68);
-		left.addCuboid("left", -12.0F, -35.0F, 4.5F, 0, 1, 1, 0.0F, 31, 68);
-		left.addCuboid("left", -12.0F, -35.0F, 6.5F, 0, 1, 1, 0.0F, 31, 68);
-		left.addCuboid("left", -12.0F, -35.0F, 8.5F, 0, 1, 1, 0.0F, 31, 68);
-		left.addCuboid("left", -12.0F, -35.0F, 10.5F, 0, 1, 1, 0.0F, 31, 68);
+public class KingTaterEntityModel extends CompositeEntityModel<KingTaterEntity> {
+	private static final ModelPart root;
+	private static final int textureWidth = 96;
+	private static final int textureHeight = 82;
+	
+	static {
+		root = new ModelPart(List.of(), Map.of(
```

Comment:
what the fuck is this formatting

Reply:
best i can do until we do entitygenjson + blockbench plugin

Reply:
wrong. do it the normal vanilla way

Reply:
> wrong. do it the normal vanilla way

this is the vanilla way right?